src/App.vue

```diff
@@ -597,6 +597,9 @@ export default {
       this.setIdentifier(this.tempRegNumber)
       this.setEntityType(legalType)
 
+      // NameRequests are always in good standing
```

Comment:
This isn't a Name Request. It's a draft application.

Reply:
Oops fixed!